### ISSUE TITLE
[4.0] Search Tools - filters [a11y]

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#34038]
the search tools filters are displayed (or hidden) using the filter options button. This uses the searchtools.js script.

There is no accessible indicator if the search tools are being displayed or not. All you can do is to select the button and then keep tabbing to see if any filters appeared. If the search tools were already displayed then you just closed them and you have to tab backwards and select the button again and then keep tabbing until you reach the buttons.

The filters should be the next item in the navigation order when they are expanded

1. the button should have aria-expanded=true/false so that you know the current state.
2. if you select the button to expand/display the filters then the next tab should take you to the first filter. and not the sort order

cosmetically it would be nice if the expand/collapse was smoothly animated (as you see when you use the bootstrap collapse for example)

**Comment by chmst** [comment]
still open #28056. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34038">issues.joomla.org/tracker/joomla-cms/34038</a>.</sub>

**Comment by brianteeman** [comment]
Thanks @chmst forgot about that and it didnt have any a11y tag or title. From re-reading it again twice that issue was more about the visual display of the search filters and not about the keyboard navigation

**Comment by Krshivam25** [comment]
@brianteeman you mean to navigate the tab order in a way that we can reach the search bar easily, right?

**Comment by brianteeman** [comment]
Yes (and this must be done naturally not by adding tabindex)

**Comment by chmst** [comment]
@brianteeman this open issue was not primary about a11y, so I did not set the label for it. Just want to keep similar things together.

**Comment by Krshivam25** [comment]
@brianteeman I think this too be fixed via tab order as I browse through tab ordering and find this :
![Screenshot (255)](https://user-images.githubusercontent.com/49578630/120319678-e71f4500-c2fe-11eb-81f5-14750aa21870.png)
